### ISSUE TITLE
Build crashes with space in filename or path

### ISSUE THREAD

**Issue by murphy2** [thomasjo/atom-latex#42]
Build crashes with space in filename or path.

**Atom Version**: 0.174.0
**System**: Mac OS X 10.10.1
**Thrown From**: [latex](https://github.com/thomasjo/atom-latex) package, v0.14.0

### Stack Trace

Uncaught Error: ENOENT, no such file or directory '/Users/Example Path/"Example File.pdf"'

```
At fs.js:580

Error: ENOENT, no such file or directory '/Users/Example Path/"Example File.pdf"'
  at Error (native)
  at Object.fs.renameSync (fs.js:580:18)
  at Object.fsPlus.moveSync (/Users/murphy/Downloads/Atom.app/Contents/Resources/app/node_modules/fs-plus/lib/fs-plus.js:221:17)
  at Object.module.exports.moveResult (/Users/murphy/.atom/packages/latex/lib/latex.coffee:123:8)
  at /Users/murphy/.atom/packages/latex/lib/latex.coffee:66:12
  at /Users/murphy/.atom/packages/latex/lib/builders/latexmk.coffee:19:9
  at ChildProcess.exithandler (child_process.js:736:7)
  at ChildProcess.emit (events.js:110:17)
  at maybeClose (child_process.js:1009:16)
  at Socket.<anonymous> (child_process.js:1177:11)
  at Socket.emit (events.js:107:17)
  at Pipe.close (net.js:469:12)

```

### Commands

```
     -0:44.4 core:move-right (atom-text-editor.editor.mini)
  8x -0:44.2 core:move-left (atom-text-editor.editor.mini)
     -0:42.7 core:backspace (atom-text-editor.editor.mini)
     -0:42.4 core:confirm (atom-text-editor.editor.mini)
  2x -0:39.3 latex:build (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:19.4 command-palette:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:17.8 core:confirm (atom-text-editor.editor.mini)
     -0:17.8 latex:sync (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:16.5 tree-view:open-selected-entry (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:11.5 core:move-right (atom-text-editor.editor.mini)
     -0:11.1 editor:move-to-beginning-of-word (atom-text-editor.editor.mini)
     -0:10.5 editor:move-to-end-of-word (atom-text-editor.editor.mini)
  8x -0:09.8 core:move-left (atom-text-editor.editor.mini)
     -0:08.2 core:confirm (atom-text-editor.editor.mini)
     -0:05.1 core:save (atom-text-editor.editor)
     -0:00.0 latex:build (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```

### Config

```json
{
  "core": {
    "themes": [
      "unity-ui",
      "base16-tomorrow-light-theme"
    ]
  }
}
```

### Installed Packages

```coffee
# User
language-latex, v0.4.1
latex, v0.14.0
unity-ui, v1.0.7

# Dev
No dev packages
```

/cc @atom/core

**Comment by thomasjo** [comment]
Thanks for reporting this! Just released v0.15.1 which should resolve your problem.

**Comment by RJC0514** [comment]
I know it's been a while but I'm having the same issue, running atom-latex 0.8.4, atom 1.25.1 on windows 10. Doesn't want to build files with spaces in their names.
